build.gradle

```diff
@@ -114,7 +114,7 @@ license {
     ext.year = Calendar.getInstance().get(Calendar.YEAR)
     ignoreFailures = true
     includes (["**/*.groovy", "**/*.java"])
-    exclude "/src/test/*"
+    excludes (["/src/test/*.groovy", "src/test/*.java"])
```

Comment:
Why was this necessary to add?

Reply:
It wasn't correctly excluding the test files previously as far as I was aware. If we had to specifically include the .java and .groovy extensions in the includes field, it would make sense to do the same for excludes.